editor.planx.uk/src/pages/FlowEditor/components/Sidebar/LinkDialog.tsx

```diff
@@ -0,0 +1,302 @@
+          {copyMessage}
+        </Typography>
+      </Button>
+    </Tooltip>
+  );
+};
+
+const InactivePublishedLink = (props: any) => {
+  return (
+    <LinkBox>
+      {" "}
+      <PaddedText variant="h4">{props.title}</PaddedText>
+      <InactiveLink pl={"31px"}>{props.message}</InactiveLink>{" "}
+    </LinkBox>
+  );
+};
+
+const ActivePublishedLink = (props: any) => {
+  return (
+    <LinkBox>
+      {" "}
+      <PaddedText variant="h4">
+        {props.title} <CopyButton link={props.link} />
+      </PaddedText>
+      <Link pl="31px" href={props.link}>
+        {props.link}
+      </Link>
+    </LinkBox>
+  );
+};
+
+const PublishedLinkContent = (props: PublishedLinkProps) => {
+  return (
+    <Stack spacing={"15px"}>
+      {props.subdomain && props.status === "online" && props.isPublished ? (
+        <ActivePublishedLink title="With subdomain" link={props.subdomain} />
+      ) : props.subdomain && props.status === "online" && !props.isPublished ? (
+        <InactivePublishedLink
+          title="With subdomain"
+          message={props.subdomain}
+        />
+      ) : props.subdomain && props.status === "offline" && props.isPublished ? (
+        <InactivePublishedLink
+          title="With subdomain"
+          message={props.subdomain}
+        />
+      ) : (
+        <InactivePublishedLink
+          title="With subdomain"
+          message={"There is not a domain configured for this team"}
+        />
+      )}
+      {props.isPublished && props.status === "online" ? (
+        <ActivePublishedLink title="PlanX link" link={props.link} />
+      ) : (
+        <InactivePublishedLink title="PlanX link" message={props.link} />
+      )}
+    </Stack>
+  );
+};
+
+const LinkContainer = (props: LinkProps) => {
+  const infoPadding = "31px";
+  return (
+    <Box display={"flex"} flexDirection={"column"} gap={"15px"} mb={1}>
+      <LinkBox>
+        <Box
+          display={"flex"}
+          flexDirection={"row"}
+          alignItems={"center"}
+          gap={"7px"}
+        >
+          <>{props.titleIcon}</>
+          <Typography variant="h4" component={"h4"}>
+            {props.title}
+          </Typography>
+          {props.type !== "published" && <CopyButton link={props.link} />}
+        </Box>
+        <Typography pl={infoPadding}>{props.description}</Typography>
+        {props.status === "offline" && props.type === "published" && (
+          <Typography pl={infoPadding}>
+            You can switch it to “Online” in your{" "}
+            <Link href={props.link.replace("/published", "/service")}>
+              Service Settings
+            </Link>
+          </Typography>
+        )}
+      </LinkBox>
+      {props.type === "published" ? (
+        <PublishedLinkContent
+          status={props.status}
+          subdomain={props.subdomain}
+          link={props.link}
+          isPublished={props.isPublished}
+        />
+      ) : (
+        <Link pl={"31px"} href={props.link}>
```

Comment:
nit: noticing lots of places throughout these file changes where `31px` is hard-coded - it'd be much more ideal to try to use an existing global `theme.spacing` value here I think ?